apps/ima/index.js

```diff
@@ -33,16 +38,90 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/evidence-upload', // TO BE UPDATED AS STEPS ARE ADDED
       backLink: 'current-progress'
     },
-    '/confirm': {
+    '/evidence-upload': {
+      behaviours: [SaveImage('image'), RemoveImage, LimitDocument],
+      fields: ['image'],
+      continueOnEdit: true,
+      next: '/are-you-submitting-this-form-late'
+    },
+    '/are-you-submitting-this-form-late': {
+      behaviours: [SaveFormSession],
+      locals: { showSaveAndExit: true },
+      fields: [
+        'are-you-submitting-this-form-late',
+        'are-you-submitting-this-form-late-extension',
+        'late-extension-options-yes-detail'
+      ],
+      continueOnEdit: true,
+      next: '/why-are-you-submitting-late'
+    },
+    '/why-are-you-submitting-late': {
+      behaviours: [SaveImage('image'), RemoveImage, LimitDocument],
+      locals: { showSaveAndExit: true },
+      fields: ['image', 'late-submission'],
+      continueOnEdit: true,
+      next: '/summary'
+    },
+    '/summary': {
       behaviours: [Summary, SaveFormSession],
       sections: require('./sections/summary-data-sections'),
       locals: { showSaveAndExit: true },
-      next: '/confirmation'
```

Comment:
this should be '/are-you-submitting-this-form-late'

Reply:
Changed to submitting-late